README.md

```diff
@@ -49,7 +49,11 @@ faster.
 When stdin is nil or an empty string, the stdin device will get closed immediately.
 
 The `timeout` parameter speifies the timeout threshold for stderr/stdout reading timeout,
-the stdin writing timeout, and the process waiting timeout, respectively.
+the stdin writing timeout, and the process waiting timeout, respectively. If the timeout
+expires, the sub-process will be killed by first a SIGTERM, and then a SIGKILL.
```

Comment:
"the timeout expires" does not make sense. Should be "the timeout threshold is exceeded"?

Also, we need to make it clear about

1. the `SIGKILL` won't be issued when `SIGTERM` already makes the process quit.
1. the waiting time between `SIGTERM` and `SIGKILL`.
1 whether we always wait the full delay between `SIGTERM` and `SIGKILL` even if the process quits almost immediately upon receiving the `SIGTERM` signal.